audit/src/main/java/tech/beshu/ror/audit/adapters/JsonObjectJavaAdapter.java

```diff
@@ -0,0 +1,27 @@
+/*
+ *    This file is part of ReadonlyREST.
+ *
+ *    ReadonlyREST is free software: you can redistribute it and/or modify
+ *    it under the terms of the GNU General Public License as published by
+ *    the Free Software Foundation, either version 3 of the License, or
+ *    (at your option) any later version.
+ *
+ *    ReadonlyREST is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU General Public License for more details.
+ *
+ *    You should have received a copy of the GNU General Public License
+ *    along with ReadonlyREST.  If not, see http://www.gnu.org/licenses/
+ */
+package tech.beshu.ror.audit.adapters;
+
+import org.json.JSONObject;
+
+@Deprecated
+class JsonObjectJavaAdapter {
```

Comment:
New constructors have appeared in the `json` lib and this caused the `ambiguous constructors` issue in our scala code. 
This adapter is needed to use the desired constructor explicitly.